content/components/banner.mdx

```diff
@@ -8,15 +8,254 @@ tags:
   - flash
 ---
 
+import {Box} from '@primer/react'
 import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 import {AccessibilityLink} from '~/src/components/accessibility-link'
 
-Banner can be used in one of two ways:
+<img
+  width="960"
+  alt="Image displaying each variant of banenr, from top: critical, warning, success, info, upsell."
+  src="https://github.com/primer/primitives/assets/813754/f96b1eda-2413-4fc1-b1a7-3775b0c3b324"
+/>
+
+## Usage
 
+Banner can be used in one of two ways:
 1. To highlight prominent information on a page.
 2. To communicate feedback in response to a user action.
 
+For system wide information, consider using an [announcement](../ui-patterns/messaging#message-types).
+
+## Anatomy
+
+<img
+  width="960"
+  alt="Anatomy of a banner, including the message, title, icon, dismiss button, primary, and secondary action"
+  src="https://github.com/primer/design/assets/813754/defa52f8-49bb-4332-aaa1-872a22a563a9"
+/>
+
+## Options
+
+### Variant
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="critical banner"
+    src="https://github.com/primer/design/assets/813754/8b135c8e-898d-423c-81c4-9dbb4736a4d5"
+  />
+  <Box as="p" mt="0">
+    <h4>Critical</h4>
+    Use critical banners to signal critical errors, system failures, or necessary destructive actions. Examples: form validation error summary, repo deletion confirmations.
```

Comment:
I would also consider using custom language for each of these variants that align with the severity.

Reply:
Can you elaborate on this, or add a suggestion? I am not sure I understand what you mean.

Reply:
Sorry, I'm referring to the images used for each case. So instead of using the same language in each image for each variant, try and be more specific.

Reply:
Just checked the deploy and I think your updates are awesome! 👍